internal/services/machinelearning/machine_learning_compute_instance_resource.go

```diff
@@ -215,10 +214,16 @@ func resourceComputeInstanceCreate(d *pluginsdk.ResourceData, meta interface{})
 		computeInstance.Properties.ComputeInstanceAuthorizationType = pointer.To(machinelearningcomputes.ComputeInstanceAuthorizationType(authType))
 	}
 
+	mlWorkspacesClient := meta.(*clients.Client).MachineLearning.Workspaces
```

Comment:
Can we move this client definition to the top of the function with the other client definitions?

Reply:
Moved